project/frontend/src/components/Authentication.js

```diff
@@ -37,9 +41,43 @@ function Authentication(props) {
     });
   }, []);
 
+  const signInWrapper = async () => {
+    let user;
+    try {
+      user = await GoogleAuth.signIn();
+    } catch (e) {
+      return;
+    }
+
+    const id = user.getId();
+    try {
+      const ref = firestore.collection('users').doc(id);
```

Comment:
do we have to await this as well?

Reply:
This line (the `.collection()` and `.doc()`) is completely synchronous so no need for await here. Only the `.get()`, `.update()`, `.delete()`, etc. on a document ref are asynchronous.

Reply:
Gotcha, thanks for explaining!